cmd/client.cc

```diff
@@ -88,7 +88,7 @@ main()
 {
     char* envVar;
 
-    TransportRemote server = TransportRemote{ "127.0.0.1", 1234, TransportProtocol::QUIC };
+    TransportRemote server = TransportRemote{ "127.0.0.1", 1234, TransportProtocol::UDP };
```

Comment:
We should probably make this a config... although I don't think it's urgent because I believe most for testing will modify this code anyway.